pkg/controllers/nodeclaim/termination/controller.go

```diff
@@ -140,9 +142,12 @@ func (c *Controller) finalize(ctx context.Context, nodeClaim *v1.NodeClaim) (rec
 			return reconcile.Result{}, client.IgnoreNotFound(fmt.Errorf("removing termination finalizer, %w", err))
 		}
 		log.FromContext(ctx).Info("deleted nodeclaim")
-		NodeClaimTerminationDuration.With(map[string]string{
+		NodeClaimTerminationDurationSeconds.With(map[string]string{
 			metrics.NodePoolLabel: nodeClaim.Labels[v1.NodePoolLabelKey],
 		}).Observe(time.Since(stored.DeletionTimestamp.Time).Seconds())
+		metrics.NodeClaimsTerminatedTotal.With(prometheus.Labels{
+			metrics.NodePoolLabel: metrics.GetLabelOrDefault(nodeClaim.Labels, v1.NodePoolLabelKey),
```

Comment:
Can we drop `GetLabelOrDefault` in favor of just returning the empty string everywhere? This is the convention for Prometheus generally rather than returning a dummy value `n/a` that represents empty